storage/innobase/lock/lock0lock.cc

```diff
@@ -1636,21 +1761,38 @@ static void lock_rec_add_to_queue(unsigned type_mode, const hash_cell_t &cell,
 	if (type_mode & LOCK_WAIT) {
 		goto create;
 	} else if (lock_t *first_lock = lock_sys_t::get_first(cell, id)) {
+		bool bypass_mode= !is_supremum
+			&& lock_t::is_rec_exclusive_not_gap(type_mode);
+		bool has_s_lock_or_stronger= false;
 		for (lock_t* lock = first_lock;;) {
```

Comment:
In the old InnoDB formatting style, which this function is following, there always was a space before `=`.

Almost the entire function is being modified here, so we might as well reformat the function in the common style.